### ISSUE TITLE
Clarifying the Full Fine-Tuning and LoRA Training Setups

### ISSUE THREAD

**Issue by yuang-lee** [Utaotao/ProFit#3]
Dear authors,

Thank you for open-sourcing this great work!

I would like to confirm whether the results reported in Table 1 were obtained using full-parameter fine-tuning, as indicated by the example command in the README and `run.sh`, which specifies:

```bash
--finetuning_type full
```

Is my understanding correct that the main results in Table 1 use full-parameter fine-tuning, and that LoRA is utilized only in Section 6.2 and Appendix C?

I would also greatly appreciate it if you could share the exact LoRA training configuration or command used for those experiments. In particular, the following details do not appear to be clearly specified in the paper or codebase:
- `lora_alpha`
- `lora_dropout`
- LoRA target modules or placement
- Whether all other training hyperparameters (learning rate, batch size) were kept identical to the full fine-tuning setup
- Whether `lora_alpha` was fixed or scaled with the LoRA rank in the rank-ablation experiments

Thank you in advance. I look forward to your clarification!
